src/Codeception/Lib/Generator/Actions.php

```diff
@@ -38,7 +38,7 @@ trait {{name}}Actions
      {{doc}}
      * @see \{{module}}::{{method}}()
      */
-    public function {{action}}({{params}}) {
+    public function {{action}}({{params}}){{return_type}} {
```

Comment:
This leads to a `Fatal error: A void function must not return a value` when your return type is void.

Reply:
See my new PR that fixes this https://github.com/Codeception/Codeception/pull/5878